src/Files.Uwp/ServicesImplementation/UpdateService.cs

```diff
@@ -158,5 +159,12 @@ private void OnUpdateCancelled()
         {
             IsUpdating = false;
         }
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        private void OnPropertyChanged([CallerMemberName] string propertyName = null)
```

Comment:
You can inherit from `: ObservableObject, ISettingsService` and drop this method